resume-buddy/src/test/java/com/google/sps/MatchTest.java

```diff
@@ -34,6 +38,7 @@
   private Entity REVIEWER_E;
   private Entity REVIEWER_F;
   private Entity REVIEWER_G;
+  private Entity USER_A;
```

Comment:
If you could change all these Entity objects to have lowercase variable names, ie `REVIEWEE_A` should be `revieweeA`, that would adhere to the Google style guide. These are normal instance variables, so they shouldn't be uppercase.